zgw/common-ground-authentication/src/main/kotlin/nl/nlportal/commonground/authentication/CommonGroundAuthentication.kt

```diff
@@ -57,24 +57,34 @@ abstract class CommonGroundAuthentication(
 
     /**
      * Gets MachtingsDienst UUIDs property from the JWT
+     * param: portalMachtigingUuid: portal uuid which indicates the user has all the machtigingen
      *
      * @return MachtingsDienst UUIDs
      */
-    fun machtigingsDienstUUIDs(): List<UUID>? {
+    fun machtigingsDienstUUIDs(allMachtigingUuid: UUID?): List<UUID>? {
```

Comment:
Can we add a default value of `null` to the `allMachtigingUuid` argument to avoid having to explicitly pass on a null.
```suggestion
    fun machtigingsDienstUUIDs(allMachtigingUuid: UUID? = null): List<UUID>? {
```

Reply:
Good point, just forgot it. Will change it